app/controllers/flood_risk_engine/enrollments/exemptions_controller.rb

```diff
@@ -0,0 +1,32 @@
+require_dependency "flood_risk_engine/application_controller"
+
+module FloodRiskEngine
+  class Enrollments::ExemptionsController < ApplicationController
+
+    def show
+      destroy # allows exemptions to be removed when JavaScript is disabled.
+    end
+
+    def destroy
+      enrollment.exemptions.destroy(exemption)
+      step_back if enrollment.exemptions.empty?
+      redirect_to enrollment_step_path(enrollment, enrollment.current_step)
+    end
+
+    private
+
+    def enrollment
```

Comment:
given its a dynamic language is there a slight performance hit using this form of always calling a method and using the cached result ? i.e rather than assigning once to a @enrollment  variable, locally  within the method ?   ..  on surface appears each subsequent call to enrollment requires couple more operations than simply using @enrollment 

Just a general question rather than a comment/suggestion (in big scheme of network etc sure its irrelevant)  .. I'm not sure how the underlying ruby compiler could/would optimise this